torch/csrc/autograd/profiler_python.cpp

```diff
@@ -204,20 +205,36 @@ struct Config<CallType::PyCall> {
   static constexpr EventType event_type = EventType::PyCall;
 };
 
-template <>
-struct Config<CallType::PyModuleCall> {
-  using key_t = PyModuleSelf;
-  using cls_t = PyModuleCls;
+template <typename Key, typename Cls, typename Info>
```

Comment:
Why not directly use typename ParameterInfo(which has the key, cls, and parameter info)? Or replace typename Info with typename ParameterInfo?

Reply:
That's a great point.